recipes/ocl-icd-system/activate.sh

```diff
@@ -0,0 +1,9 @@
+conda_ocl_icd_system_activate () {
+    # If no OpenCL ICD has been installed with conda yet
+    if [[ -z "$(ls ${CONDA_PREFIX}/etc/OpenCL/vendors/)" ]]; then
+        ln -s /etc/OpenCL/vendors "${CONDA_PREFIX}"/etc/OpenCL/vendors/system
+        touch "${CONDA_PREFIX}"/etc/OpenCL/vendors/.ocl_icd_system_cf
```

Comment:
Why this ?

Reply:
To make sure it was created by this package, and not by the user. It would be very unlucky, but possible?

Reply:
Why not do change the name from `system` to `system_cf` or something similar.

Reply:
What about `ocl-icd-system` so it is consistent with the package name?

Reply:
Sure.